pydata_center/predictive/predictor.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import numpy as np
+from predictive.models.model_io import load_model
+
+
+class Predictor:
+    def __init__(self):
+        model_dir = os.path.join(os.path.dirname(__file__), 'models')
+        self.cpu_model = load_model(
+            os.path.join(model_dir, 'cpu_forecast.pkl')
+        )
+        self.iso_model = load_model(os.path.join(model_dir, 'iso_anomaly.pkl'))
```

Comment:
Hardcoding the model directory path could be improved by using configuration or environment variables.